osm2gtfs/creators/et_addisababa/trips_creator_et_addisababa.py

```diff
@@ -0,0 +1,133 @@
+# coding=utf-8
+
+from datetime import timedelta, datetime
+
+from osm2gtfs.creators.trips_creator import TripsCreator
+from osm2gtfs.core.helper import Helper
+from osm2gtfs.core.elements import Line
+
+from transitfeed.trip import Trip
+
+ 
+def time_string_to_minutes(time_string):
+    (hours, minutes, seconds) = time_string.split(':')
+    return int(hours) * 60 + int(minutes)
+
+class TripsCreatorEtAddisababa(TripsCreator):
+    service_weekday = None
+
+    def add_trips_to_feed(self, feed, data):
+        self.service_weekday = feed.GetDefaultServicePeriod()
+        self.service_weekday.SetStartDate(
+            self.config['feed_info']['start_date'])
+        self.service_weekday.SetEndDate(self.config['feed_info']['end_date'])
+        self.service_weekday.SetWeekdayService(True)
+        self.service_weekday.SetWeekendService(True)
+
+        lines = data.routes
+        for route_ref, line in sorted(lines.iteritems()):
+            if not isinstance(line, Line):
+                continue
+            print("Generating schedule for line: " + route_ref)
+
+            flex_flag = None
+            if 'route_master' in line.tags and line.tags['route_master'] == "light_rail":
+                route_type = "Tram"
```

Comment:
I don't understand why you need to override the route_type here :thinking:

Reply:
In OSM we tagged it as light_rail, and I was thinking it needs to be mapped to Tram in GTFS; but I am not sure